ksql-engine/src/test/java/io/confluent/ksql/EndToEndEngineTestUtil.java

```diff
@@ -779,10 +786,42 @@ private static TopologyAndConfigs readTopologyFile(final String file, final Obje
     }
   }
 
-  static Stream<JsonTestCase> findTestCases(final Path dir) {
+  private static List<Path> getTests(final Path dir, final List<String> files) {
+    List<Path> filePaths = new ArrayList<>();
```

Comment:
Could this be:
`return files.stream().map(name -> dir.resolve(name.trim())).collect(Collectors.toList());`

Reply:
Thanks. Done.